docs/application/dotnet/learn-about-dotnet.md

```diff
@@ -32,7 +32,10 @@ Managed runtime offers the following advantages to your application development:
 
 ## Tizen .NET Features and Components
```

Comment:
For second heading, please use sentence style capitalization
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
## Tizen .NET features and components
```

Reply:
Hello @omar-srbd . This line was not changed by this commit. I can apply your suggestion for overall improvement but it will be great to fix only changed lines...